src/App.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+
+import Header from "./components/Header";
+// import Features from "./components/Features";
+// import Preview from "./components/Preview";
+// import GetStarted from "./components/GetStarted";
+// import MoreOSS from "./components/MoreOSS";
+// import Footer from "./components/Footer";
+
+const Container = styled.div`
+  width: 100%;
+`;
+
+const config = {
```

Comment:
This is an example config using the content from Renature docs. Is this a good approach to the components?

Reply:
This lgtm.. exactly the kind of thing I was imagining!

Reply:
Yes! I think this is great